frontend/src/core/utils/fluent/isPluralElement.test.js

```diff
@@ -0,0 +1,65 @@
+import isPluralElement from './isPluralElement';
+import parser from './parser';
+
+
+describe('isPluralElement', () => {
+    it('returns false for elements that are not select expressions', () => {
+        const input = 'my-entry = Hello!';
+        const message = parser.parseEntry(input);
+        const element = message.value.elements[0];
+
+        expect(isPluralElement(element)).toEqual(false);
+    });
+
+    it('returns true if all variant keys are CLDR plurals', () => {
+        const input = `
+my-entry =
+    { $num ->
+        [one] Hello!
+       *[two] World!
+    }`;
+        const message = parser.parseEntry(input);
+        const element = message.value.elements[0];
+
+        expect(isPluralElement(element)).toEqual(true);
+    });
+
+    it('returns true if all variant keys are numbers', () => {
+        const input = `
+my-entry =
+    { $num ->
+        [1] Hello!
+       *[2] World!
+    }`;
```

Comment:
Is this the current behavior? I don't think that should be the case. In practice, this shouldn't be the case to begin with, but this is more likely a 1 vs many than an actual plural.